xml/Microsoft.Extensions.FileProviders.Physical/PhysicalDirectoryInfo.xml

```diff
@@ -112,7 +112,8 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>
+          <see langword="true" /> if resource exists in the underlying storage system.</summary>
         <value>To be added.</value>
```

Comment:
Language can be improved, and the information should be split between the summary (description of the property) and the value (the possible values the property can return):
```suggestion
        <summary>Determines if the resource exists in the underlying storage system.</summary>
        <value><see langword="true" /> if the resource exists in the underlying storage system; <see langword="false" /> otherwise.</value>
```